src/modules/RootPart.rb

```diff
@@ -1240,6 +1240,25 @@ def MountFSTab(fstab, message)
         )
       end
 
+      efivars_path = "/sys/firmware/efi/efivars"
+      if ::File.exist?(efivars_path)
```

Comment:
Could we simply merge these conditions? `if ::File.exist?(efivars_path) && MountPartition(...`

Reply:
IMHO bad style because the first is a condition and the second is a side effect (with a result)